.github/workflows/deploy-to-gh-pages.yml

```diff
@@ -8,15 +8,23 @@ on:
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
+
     env:
       MAPBOX_ACCESS_TOKEN: ${{ secrets.MAPBOX_ACCESS_TOKEN }}
       SITE_NAME: 'Catalyst AIP'
+
+    strategy:
+      matrix:
+        node-version: [10, 12, 14]
```

Comment:
This sounds fine to me, but does it slow down the build?

Reply:
I guess it runs 3 times as long compared to running the workflow with only one node version. I just thought it was cool to test for more versions, since it was so easily available 😎 If we feel slowed down by this setting, we can remove it. 

Overall, there is definitely room for improvement in the way I set up the Github Actions.

Reply:
Oh, now after merging I see what you mean: not only does it slow down the build, it is going to deploy 3 times in a row 🙈